src/libsyntax/codemap.rs

```diff
@@ -572,6 +559,20 @@ impl CodeMapper for CodeMap {
         }
         sp
     }
+    fn load_source_for_filemap(&self, filename: FileName) -> bool {
+        let file_map = if let Some(fm) = self.get_filemap(&filename) {
```

Comment:
I don't think this is a good idea - a caller would already have the `FileMap`, and *not* from a string lookup, right?